spec/features/fill_in_listing_form_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'listing_web_helper'
+
+feature 'filling in form' do
+
+  scenario 'user can fill in form' do
+    create_listing('My space')
```

Comment:
Is this a test? Not seeing an expect statement so not sure if this is actually doing something

Reply:
This is just testing the form works - kind of redundant with the other tests now though, so may be worth removing